### PR TITLE
[upload] - fix an issue where json content items were not unified with -z

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5593)

## Must have
- [x] Tests

### CODE REVIEW

**Comment by GuyAfik** [comment]
1. The changes in `repo.py` is a change because graph content objects validations do not pass with old values
2. The changes in a lot of test files are additions of some values because they are failing on graph validations.

File: demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py [hunk 1]
```diff
@@ -120,7 +123,21 @@ def dump(
...
+        try:
+            created_files.extend(
+                self._unify(
+                    dest_dir=self._create_target_dump_dir(dest_dir=dest_dir),
+                    output=self.normalize_file_name(),
+                )
+            )
+            logger.debug(f"Successfully unified {self.path} {self.type()}")
+        except Exception as e:
+            logger.debug(
```
Comment (dorschw) [inline demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py:136]:
```suggestion
            logger.error(
```

File: demisto_sdk/tests/integration_tests/upload_integration_test.py [hunk 2]
```diff
@@ -91,6 +91,66 @@ def test_integration_upload_pack_positive(demisto_client, mocker, repo):
...
+    pack = repo.setup_one_pack(name="test-pack")
+
+    mocker.patch(
+        "demisto_sdk.commands.upload.uploader.Uploader.notify_user_should_override_packs",
+        return_value=True,
+    )
+
+    mocker.patch(
+        "demisto_sdk.commands.common.content.objects.pack_objects.pack.get_demisto_version",
+        return_value=parse("6.5.0"),
```
Comment (dorschw) [inline demisto_sdk/tests/integration_tests/upload_integration_test.py:116]: consider using a constant, if applicable

**Review by dorschw — APPROVED** [review]

**Review by ilaner — APPROVED** [review]

File: demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py [hunk 3]
```diff
@@ -120,7 +123,21 @@ def dump(
...
+        try:
+            created_files.extend(
+                self._unify(
+                    dest_dir=self._create_target_dump_dir(dest_dir=dest_dir),
+                    output=self.normalize_file_name(),
+                )
+            )
+            logger.debug(f"Successfully unified {self.path} {self.type()}")
+        except Exception as e:
+            logger.error(
```
Comment (dantavori) [inline demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py:136]: maybe debug? it's not necessarily an error, correct me if I'm wrong

Comment (GuyAfik) [reply demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py:136]: I agree, can be changed to debug, but @dorschw thinks it should be debug log.

Comment (dorschw) [reply demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py]: If it's not an error, and users won't feel it, let it be `debug`

**Pull request merged**